--- a/config.inc.php	2016-05-22 04:06:46.000000000 -0700
+++ b/config.inc.php	2016-08-26 15:22:25.889224000 -0700
@@ -12,11 +12,11 @@
 
 // Require the new password to be a certain length.
 // set to blank to allow passwords of any length
-$config['password_minimum_length'] = 0;
+$config['password_minimum_length'] = 8;
 
 // Require the new password to contain a letter and punctuation character
 // Change to false to remove this check.
-$config['password_require_nonalpha'] = false;
+$config['password_require_nonalpha'] = true;
 
 // Enables logging of password changes into logs/password
 $config['password_log'] = false;
@@ -41,7 +41,7 @@
 // Possible options: des-crypt, ext-des-crypt, md5-crypt, blowfish-crypt,
 // sha256-crypt, sha512-crypt, md5, sha, smd5, ssha, samba, ad, dovecot, clear.
 // For details see password::hash_password() method.
-$config['password_algorithm'] = 'clear';
+$config['password_algorithm'] = 'blowfish-crypt';
 
 // Password prefix (e.g. {CRYPT}, {SHA}) for passwords generated
 // using password_algorithm above. Default: empty.
@@ -78,7 +78,7 @@
 // ------------------
 // PEAR database DSN for performing the query. By default
 // Roundcube DB settings are used.
-$config['password_db_dsn'] = '';
+$config['password_db_dsn'] = 'pgsql://roundcube:RoundCubePassword@localhost/mail';
 
 // The SQL query used to change the password.
 // The query can contain the following macros that will be expanded as follows:
@@ -103,12 +103,12 @@
 //      %q is replaced with the hashed password before the change
 // Escaping of macros is handled by this module.
 // Default: "SELECT update_passwd(%c, %u)"
-$config['password_query'] = 'SELECT update_passwd(%c, %u)';
+$config['password_query'] = 'UPDATE mailboxes SET password = %P, modified = now() WHERE address = %u';
 
 // By default the crypt() function which is used to create the %c
 // parameter uses the md5 algorithm (deprecated, use %P).
 // You can choose between: des, md5, blowfish, sha256, sha512.
-$config['password_crypt_hash'] = 'md5';
+$config['password_crypt_hash'] = 'blowfish';
 
 // By default domains in variables are using unicode.
 // Enable this option to use punycoded names
