--- a/system	2016-08-13 05:14:35.924569000 -0700
+++ b/system	2016-08-12 06:28:12.056944000 -0700
@@ -9,7 +9,7 @@
 auth		requisite	pam_opieaccess.so	no_warn allow_local
 #auth		sufficient	pam_krb5.so		no_warn try_first_pass
 #auth		sufficient	pam_ssh.so		no_warn try_first_pass
-auth		required	pam_unix.so		no_warn try_first_pass nullok
+auth		required	pam_unix.so		no_warn try_first_pass nullok authtok_prompt="Password: "
 
 # account
 #account	required	pam_krb5.so
