--- a/login.conf	2021-07-03 15:44:44.154187000 -0700
+++ b/login.conf	2021-07-03 15:45:22.948238000 -0700
@@ -26,7 +26,7 @@
 	:passwd_format=sha512:\
 	:copyright=/etc/COPYRIGHT:\
 	:welcome=/var/run/motd:\
-	:setenv=BLOCKSIZE=K:\
+	:setenv=BLOCKSIZE=K,LC_COLLATE=C:\
 	:mail=/var/mail/$:\
 	:path=/sbin /bin /usr/sbin /usr/bin /usr/local/sbin /usr/local/bin ~/bin:\
 	:nologin=/var/run/nologin:\
@@ -49,7 +49,7 @@
 	:ignoretime@:\
 	:umask=022:\
 	:charset=UTF-8:\
-	:lang=C.UTF-8:
+	:lang=en_US.UTF-8:
 
 #
 # A collection of common class names - forward them all to 'default'
