diff -ru a/postgresql.conf b/postgresql.conf
--- a/postgresql.conf	2019-10-20 17:29:38.956271000 -0700
+++ b/postgresql.conf	2019-10-20 17:31:33.525534000 -0700
@@ -60,7 +60,7 @@
 					# comma-separated list of addresses;
 					# defaults to 'localhost'; use '*' for all
 					# (change requires restart)
-#port = 5432				# (change requires restart)
+port = 5432				# (change requires restart)
 max_connections = 100			# (change requires restart)
 #superuser_reserved_connections = 3	# (change requires restart)
 #unix_socket_directories = '/tmp'	# comma-separated list of directories
