diff -ru a/postgresql.conf b/postgresql.conf
--- a/postgresql.conf	2026-04-18 11:32:29.311441000 -0700
+++ b/postgresql.conf	2026-04-18 11:33:18.891542000 -0700
@@ -61,7 +61,7 @@
 					# comma-separated list of addresses;
 					# defaults to 'localhost'; use '*' for all
 					# (change requires restart)
-#port = 5432				# (change requires restart)
+port = 5432				# (change requires restart)
 max_connections = 100			# (change requires restart)
 #reserved_connections = 0		# (change requires restart)
 #superuser_reserved_connections = 3	# (change requires restart)
