diff --git a/lib/system_check/rake_task/app_task.rb b/lib/system_check/rake_task/app_task.rb
index 8e711138856f..29116dffe6d2 100644
--- a/lib/system_check/rake_task/app_task.rb
+++ b/lib/system_check/rake_task/app_task.rb
@@ -25,8 +25,8 @@ def self.checks
           SystemCheck::App::UploadsDirectoryExistsCheck,
           SystemCheck::App::UploadsPathPermissionCheck,
           SystemCheck::App::UploadsPathTmpPermissionCheck,
-          SystemCheck::App::SystemdUnitFilesOrInitScriptExistCheck,
-          SystemCheck::App::SystemdUnitFilesOrInitScriptUpToDateCheck,
+          #SystemCheck::App::SystemdUnitFilesOrInitScriptExistCheck,
+          #SystemCheck::App::SystemdUnitFilesOrInitScriptUpToDateCheck,
           SystemCheck::App::ProjectsHaveNamespaceCheck,
           SystemCheck::App::RedisVersionCheck,
           SystemCheck::App::RubyVersionCheck,
diff --git a/lib/system_check/sidekiq_check.rb b/lib/system_check/sidekiq_check.rb
index 8b53182f8856..1847262d6d0c 100644
--- a/lib/system_check/sidekiq_check.rb
+++ b/lib/system_check/sidekiq_check.rb
@@ -17,7 +17,7 @@ def multi_check
     def check_sidekiq_running
       $stdout.print "Running? ... "
 
-      if sidekiq_worker_process_count > 0
+      if sidekiq_worker_process_count > 0 || sidekiq_cluster_process_count > 0
         $stdout.puts Rainbow("yes").green
       else
         $stdout.puts Rainbow("no").red
