diff --git a/lib/system_check/rake_task/app_task.rb b/lib/system_check/rake_task/app_task.rb
index 20332d4b24ba..bfc864068a37 100644
--- a/lib/system_check/rake_task/app_task.rb
+++ b/lib/system_check/rake_task/app_task.rb
@@ -24,8 +24,8 @@ def self.checks
           SystemCheck::App::UploadsDirectoryExistsCheck,
           SystemCheck::App::UploadsPathPermissionCheck,
           SystemCheck::App::UploadsPathTmpPermissionCheck,
-          SystemCheck::App::SystemdUnitFilesOrInitScriptExistCheck,
-          SystemCheck::App::SystemdUnitFilesOrInitScriptUpToDateCheck,
+          #SystemCheck::App::SystemdUnitFilesOrInitScriptExistCheck,
+          #SystemCheck::App::SystemdUnitFilesOrInitScriptUpToDateCheck,
           SystemCheck::App::ProjectsHaveNamespaceCheck,
           SystemCheck::App::RedisVersionCheck,
           SystemCheck::App::RubyVersionCheck,
diff --git a/lib/system_check/sidekiq_check.rb b/lib/system_check/sidekiq_check.rb
index 777e06f7501f..461b64248721 100644
--- a/lib/system_check/sidekiq_check.rb
+++ b/lib/system_check/sidekiq_check.rb
@@ -17,7 +17,7 @@ def multi_check
     def check_sidekiq_running
       $stdout.print "Running? ... "
 
-      if sidekiq_worker_process_count > 0
+      if sidekiq_worker_process_count > 0 || sidekiq_cluster_process_count > 0
         $stdout.puts "yes".color(:green)
       else
         $stdout.puts "no".color(:red)
