diff --git a/lib/system_check/rake_task/app_task.rb b/lib/system_check/rake_task/app_task.rb
index f7d2bf86c784..69086139c12e 100644
--- a/lib/system_check/rake_task/app_task.rb
+++ b/lib/system_check/rake_task/app_task.rb
@@ -23,8 +23,8 @@ def self.checks
           SystemCheck::App::UploadsDirectoryExistsCheck,
           SystemCheck::App::UploadsPathPermissionCheck,
           SystemCheck::App::UploadsPathTmpPermissionCheck,
-          SystemCheck::App::InitScriptExistsCheck,
-          SystemCheck::App::InitScriptUpToDateCheck,
+          #SystemCheck::App::InitScriptExistsCheck,
+          #SystemCheck::App::InitScriptUpToDateCheck,
           SystemCheck::App::ProjectsHaveNamespaceCheck,
           SystemCheck::App::RedisVersionCheck,
           SystemCheck::App::RubyVersionCheck,
diff --git a/lib/system_check/sidekiq_check.rb b/lib/system_check/sidekiq_check.rb
index 7ac1bd58edee..1b5cb330bcf9 100644
--- a/lib/system_check/sidekiq_check.rb
+++ b/lib/system_check/sidekiq_check.rb
@@ -15,7 +15,7 @@ def multi_check
     def check_sidekiq_running
       $stdout.print "Running? ... "
 
-      if sidekiq_worker_process_count > 0
+      if sidekiq_worker_process_count > 0 || sidekiq_cluster_process_count > 0
         $stdout.puts "yes".color(:green)
       else
         $stdout.puts "no".color(:red)
