diff --git a/bazel/grpc_deps.bzl b/bazel/grpc_deps.bzl
index a982d8079b5a..ec4f43ce6c1a 100644
--- a/bazel/grpc_deps.bzl
+++ b/bazel/grpc_deps.bzl
@@ -343,11 +343,10 @@ def grpc_deps():
     if "com_google_absl" not in native.existing_rules():
         http_archive(
             name = "com_google_absl",
-            sha256 = "59d2976af9d6ecf001a81a35749a6e551a335b949d34918cfade07737b9d93c5",
-            strip_prefix = "abseil-cpp-20230802.0",
+            sha256 = "987ce98f02eefbaf930d6e38ab16aa05737234d7afbab2d5c4ea7adbe50c28ed",
+            strip_prefix = "abseil-cpp-20230802.1",
             urls = [
-                "https://storage.googleapis.com/grpc-bazel-mirror/github.com/abseil/abseil-cpp/archive/20230802.0.tar.gz",
-                "https://github.com/abseil/abseil-cpp/archive/20230802.0.tar.gz",
+                "https://github.com/abseil/abseil-cpp/archive/20230802.1.tar.gz",
             ],
         )
 
diff --git a/templates/gRPC-C++.podspec.template b/templates/gRPC-C++.podspec.template
index 739e2af928d8..412047ac97b8 100644
--- a/templates/gRPC-C++.podspec.template
+++ b/templates/gRPC-C++.podspec.template
@@ -174,7 +174,7 @@
       ss.header_mappings_dir = '.'
       ss.dependency "#{s.name}/Interface", version
       ss.dependency 'gRPC-Core', version
-      abseil_version = '1.20230802.0'
+      abseil_version = '1.20230802.1'
       % for abseil_spec in grpcpp_abseil_specs:
       ss.dependency '${abseil_spec}', abseil_version
       % endfor
diff --git a/templates/gRPC-Core.podspec.template b/templates/gRPC-Core.podspec.template
index 626e67b4a706..e5dcf5775e8a 100644
--- a/templates/gRPC-Core.podspec.template
+++ b/templates/gRPC-Core.podspec.template
@@ -122,7 +122,7 @@
     s.requires_arc = false
 
     name = 'grpc'
-    abseil_version = '1.20230802.0'
+    abseil_version = '1.20230802.1'
 
     # When creating a dynamic framework, name it grpc.framework instead of gRPC-Core.framework.
     # This lets users write their includes like `#include <grpc/grpc.h>` as opposed to `#include
diff --git a/third_party/abseil-cpp b/third_party/abseil-cpp
index 29bf8085f3bf..fb3621f4f897 160000
--- a/third_party/abseil-cpp
+++ b/third_party/abseil-cpp
@@ -1 +1 @@
-Subproject commit 29bf8085f3bf17b84d30e34b3d7ff8248fda404e
+Subproject commit fb3621f4f897824c0dbe0615fa94543df6192f30
diff --git a/tools/run_tests/sanity/check_submodules.sh b/tools/run_tests/sanity/check_submodules.sh
index 38b1cb4e201c..49f2d579148a 100755
--- a/tools/run_tests/sanity/check_submodules.sh
+++ b/tools/run_tests/sanity/check_submodules.sh
@@ -25,7 +25,7 @@ want_submodules=$(mktemp /tmp/submXXXXXX)
 
 git submodule | sed 's/+//g' | awk '{ print $2 " " $1 }' | sort >"$submodules"
 cat <<EOF | sort >"$want_submodules"
-third_party/abseil-cpp 29bf8085f3bf17b84d30e34b3d7ff8248fda404e
+third_party/abseil-cpp fb3621f4f897824c0dbe0615fa94543df6192f30
 third_party/benchmark 015d1a091af6937488242b70121858bce8fd40e9
 third_party/bloaty 60209eb1ccc34d5deefb002d1b7f37545204f7f2
 third_party/boringssl-with-bazel 2ff4b968a7e0cfee66d9f151cb95635b43dc1d5b
