diff -ru a/conf.d/10-master.conf b/conf.d/10-master.conf
--- a/conf.d/10-master.conf	2020-01-20 23:46:55.328786000 -0800
+++ b/conf.d/10-master.conf	2020-01-20 23:47:38.045078000 -0800
@@ -56,6 +56,13 @@
     #mode = 0666
   }
 
+  # Expose an LMTP socket for postfix to deliver mail.
+  unix_listener /var/spool/postfix/private/dovecot-lmtp {
+    mode = 0600
+    user = postfix
+    group = postfix
+  }
+
   # Create inet listener only if you can't use the above UNIX socket
   #inet_listener lmtp {
     # Avoid making LMTP visible for the entire internet
@@ -104,9 +111,11 @@
   }
 
   # Postfix smtp-auth
-  #unix_listener /var/spool/postfix/private/auth {
-  #  mode = 0666
-  #}
+  unix_listener /var/spool/postfix/private/auth {
+    mode = 0660
+    user = postfix
+    group = postfix
+  }
 
   # Auth process is run as this user.
   #user = $default_internal_user
