diff -ru a/conf.d/10-auth.conf b/conf.d/10-auth.conf
--- a/conf.d/10-auth.conf	2020-01-06 07:54:20.545568000 -0800
+++ b/conf.d/10-auth.conf	2020-01-17 22:33:41.829683000 -0800
@@ -7,7 +7,7 @@
 # matches the local IP (ie. you're connecting from the same computer), the
 # connection is considered secure and plaintext authentication is allowed.
 # See also ssl=required setting.
-#disable_plaintext_auth = yes
+disable_plaintext_auth = yes
 
 # Authentication cache size (e.g. 10M). 0 means it's disabled. Note that
 # bsdauth, PAM and vpopmail require cache_key to be set for caching to be used.
@@ -48,7 +48,7 @@
 # the standard variables here, eg. %Lu would lowercase the username, %n would
 # drop away the domain if it was given, or "%n-AT-%d" would change the '@' into
 # "-AT-". This translation is done after auth_username_translation changes.
-#auth_username_format = %Lu
+auth_username_format = %Lu
 
 # If you want to allow master users to log in by specifying the master
 # username within the normal username string (ie. not using SASL mechanism's
@@ -97,7 +97,7 @@
 #   plain login digest-md5 cram-md5 ntlm rpa apop anonymous gssapi otp skey
 #   gss-spnego
 # NOTE: See also disable_plaintext_auth setting.
-auth_mechanisms = plain
+auth_mechanisms = plain login
 
 ##
 ## Password and user databases
@@ -119,9 +119,9 @@
 #!include auth-deny.conf.ext
 #!include auth-master.conf.ext
 
-!include auth-system.conf.ext
+#!include auth-system.conf.ext
 #!include auth-sql.conf.ext
-#!include auth-ldap.conf.ext
+!include auth-ldap.conf.ext
 #!include auth-passwdfile.conf.ext
 #!include auth-checkpassword.conf.ext
 #!include auth-vpopmail.conf.ext
diff -ru a/conf.d/10-mail.conf b/conf.d/10-mail.conf
--- a/conf.d/10-mail.conf	2020-01-06 07:54:20.545781000 -0800
+++ b/conf.d/10-mail.conf	2020-01-17 22:37:24.031715000 -0800
@@ -27,7 +27,7 @@
 #
 # <doc/wiki/MailLocation.txt>
 #
-#mail_location = 
+mail_location = mdbox:%{home}/mail
 
 # If you need to set multiple mailbox locations or want to change default
 # namespace settings, you can do it by defining namespace sections.
@@ -46,7 +46,7 @@
   # Hierarchy separator to use. You should use the same separator for all
   # namespaces or some clients get confused. '/' is usually a good one.
   # The default however depends on the underlying mail storage format.
-  #separator = 
+  separator = /
 
   # Prefix required to access this namespace. This needs to be different for
   # all namespaces. For example "Public/".
@@ -105,13 +105,13 @@
 # System user and group used to access mails. If you use multiple, userdb
 # can override these by returning uid or gid fields. You can use either numbers
 # or names. <doc/wiki/UserIds.txt>
-#mail_uid =
-#mail_gid =
+mail_uid = virtual_mail
+mail_gid = virtual_mail
 
 # Group to enable temporarily for privileged operations. Currently this is
 # used only with INBOX when either its initial creation or dotlocking fails.
 # Typically this is set to "mail" to give access to /var/mail.
-#mail_privileged_group =
+mail_privileged_group = virtual_mail
 
 # Grant access to these supplementary groups for mail processes. Typically
 # these are used to set up access to shared mailboxes. Note that it may be
@@ -158,7 +158,9 @@
 #   optimized (default): Whenever necessary to avoid losing important data
 #   always: Useful with e.g. NFS when write()s are delayed
 #   never: Never use it (best performance, but crashes can lose data)
-#mail_fsync = optimized
+# To improve performance, disable fsync globally; we will enable it for
+# some specific services later on.
+mail_fsync = never
 
 # Locking method for index files. Alternatives are fcntl, flock and dotlock.
 # Dotlocking uses some tricks which may create more disk I/O than other locking
@@ -175,7 +177,7 @@
 # to make sure that users can't log in as daemons or other system users.
 # Note that denying root logins is hardcoded to dovecot binary and can't
 # be done even if first_valid_uid is set to 0.
-#first_valid_uid = 500
+first_valid_uid = 100
 #last_valid_uid = 0
 
 # Valid GID range for users, defaults to non-root/wheel. Users having
@@ -386,11 +388,11 @@
 # this for now.
 
 # Directory root where to store mail attachments. Disabled, if empty.
-#mail_attachment_dir =
+mail_attachment_dir = /var/mail/attachments
 
 # Attachments smaller than this aren't saved externally. It's also possible to
 # write a plugin to disable saving specific attachments externally.
-#mail_attachment_min_size = 128k
+mail_attachment_min_size = 64k
 
 # Filesystem backend to use for saving attachments:
 #  posix : No SiS done by Dovecot (but this might help FS's own deduplication)
diff -ru a/conf.d/10-master.conf b/conf.d/10-master.conf
--- a/conf.d/10-master.conf	2020-01-06 07:54:20.545846000 -0800
+++ b/conf.d/10-master.conf	2020-01-17 22:33:41.830121000 -0800
@@ -116,7 +116,7 @@
   # Auth worker process is run as root by default, so that it can access
   # /etc/shadow. If this isn't necessary, the user should be changed to
   # $default_internal_user.
-  #user = root
+  user = virtual_mail
 }
 
 service dict {
diff -ru a/conf.d/10-ssl.conf b/conf.d/10-ssl.conf
--- a/conf.d/10-ssl.conf	2020-01-06 07:54:20.545913000 -0800
+++ b/conf.d/10-ssl.conf	2020-01-17 22:33:41.830297000 -0800
@@ -3,14 +3,14 @@
 ##
 
 # SSL/TLS support: yes, no, required. <doc/wiki/SSL.txt>
-#ssl = yes
+ssl = required
 
 # PEM encoded X.509 SSL/TLS certificate and private key. They're opened before
 # dropping root privileges, so keep the key file unreadable by anyone but
 # root. Included doc/mkcert.sh can be used to easily generate self-signed
 # certificate, just make sure to update the domains in dovecot-openssl.cnf
-ssl_cert = </etc/ssl/certs/dovecot.pem
-ssl_key = </etc/ssl/private/dovecot.pem
+ssl_cert = </etc/ssl/certs/foo.com_wildcard.pem
+ssl_key = </etc/ssl/private/foo.com_wildcard.key
 
 # If key file is password protected, give the password here. Alternatively
 # give it when starting dovecot with -p parameter. Since this file is often
@@ -51,7 +51,7 @@
 # Generate new params with `openssl dhparam -out /usr/local/etc/dovecot/dh.pem 4096`
 # Or migrate from old ssl-parameters.dat file with the command dovecot
 # gives on startup when ssl_dh is unset.
-#ssl_dh = </usr/local/etc/dovecot/dh.pem
+ssl_dh = </usr/local/etc/dovecot/dh.pem
 
 # Minimum SSL protocol version to use. Potentially recognized values are SSLv3,
 # TLSv1, TLSv1.1, and TLSv1.2, depending on the OpenSSL version used.
diff -ru a/conf.d/15-lda.conf b/conf.d/15-lda.conf
--- a/conf.d/15-lda.conf	2020-01-06 07:54:20.545976000 -0800
+++ b/conf.d/15-lda.conf	2020-01-17 22:38:12.273553000 -0800
@@ -43,6 +43,9 @@
 #lda_mailbox_autosubscribe = no
 
 protocol lda {
+  # Use fsync for write-safety: this deals with delivering actual mail.
+  mail_fsync = optimized
+  
   # Space separated list of plugins to load (default is global mail_plugins).
   #mail_plugins = $mail_plugins
 }
diff -ru a/conf.d/15-mailboxes.conf b/conf.d/15-mailboxes.conf
--- a/conf.d/15-mailboxes.conf	2020-01-06 07:54:20.545913000 -0800
+++ b/conf.d/15-mailboxes.conf	2020-01-17 22:33:41.830297000 -0800
@@ -49,18 +49,30 @@
   # These mailboxes are widely used and could perhaps be created automatically:
   mailbox Drafts {
     special_use = \Drafts
+    auto = subscribe
   }
+
+  # For \Junk mailboxes, there are two widely used names. We'll mark both of
+  # them as \Junk, but only the one preferred by our webmail as auto subscribe.
+  mailbox Spam {
+    special_use = \Junk
+    auto = subscribe
+  }
   mailbox Junk {
     special_use = \Junk
   }
+
   mailbox Trash {
     special_use = \Trash
+    auto = subscribe
   }
 
   # For \Sent mailboxes there are two widely used names. We'll mark both of
-  # them as \Sent. User typically deletes one of them if duplicates are created.
+  # them as \Sent, but only the one preferred by our webmail as auto subscribe.
+  # User typically deletes one of them if duplicates are created.
   mailbox Sent {
     special_use = \Sent
+    auto = subscribe
   }
   mailbox "Sent Messages" {
     special_use = \Sent
diff -ru a/conf.d/20-imap.conf b/conf.d/20-imap.conf
--- a/conf.d/20-imap.conf	2020-01-06 07:54:20.546107000 -0800
+++ b/conf.d/20-imap.conf	2020-01-17 22:39:17.836595000 -0800
@@ -39,7 +39,7 @@
 
 # How long to wait between "OK Still here" notifications when client is
 # IDLEing.
-#imap_idle_notify_interval = 2 mins
+imap_idle_notify_interval = 15 mins
 
 # ID field names and values to send to clients. Using * as the value makes
 # Dovecot use the default value. The following fields have default values
diff -ru a/conf.d/20-lmtp.conf b/conf.d/20-lmtp.conf
--- a/conf.d/20-lmtp.conf	2020-01-06 07:54:20.546178000 -0800
+++ b/conf.d/20-lmtp.conf	2020-01-17 22:39:34.673621000 -0800
@@ -35,6 +35,9 @@
 #lmtp_client_workarounds =
 
 protocol lmtp {
+  # Use fsync for write-safety: this deals with delivering actual mail.
+  mail_fsync = optimized
+  
   # Space separated list of plugins to load (default is global mail_plugins).
   #mail_plugins = $mail_plugins
 }
diff -ru a/conf.d/auth-ldap.conf.ext b/conf.d/auth-ldap.conf.ext
--- a/conf.d/auth-ldap.conf.ext	2020-01-06 07:54:20.545108000 -0800
+++ b/conf.d/auth-ldap.conf.ext	2020-01-17 22:33:41.830449000 -0800
@@ -16,12 +16,17 @@
 #  driver = prefetch
 #}
 
+# Using the LDAP driver for userdb because it's the only way we can
+# have a functional iterate_filter, which is required for using the
+# administrative commands from doveadm that act on all mailboxes.
+# If it weren't for that, we could easily have gone with with the
+# static driver for userdb.
 userdb {
   driver = ldap
-  args = /usr/local/etc/dovecot/dovecot-ldap.conf.ext
+  args = /usr/local/etc/dovecot/dovecot-ldap-userdb.conf.ext
   
   # Default fields can be used to specify defaults that LDAP may override
-  #default_fields = home=/home/virtual/%u
+  default_fields = uid=virtual_mail gid=virtual_mail home=/var/mail/virtual/%d/%n
 }
 
 # If you don't have any user-specific settings, you can avoid the userdb LDAP
diff -ru a/dovecot-ldap.conf.ext b/dovecot-ldap.conf.ext
--- a/dovecot-ldap.conf.ext	2020-01-06 07:54:20.555122000 -0800
+++ b/dovecot-ldap.conf.ext	2020-01-17 22:33:41.830640000 -0800
@@ -17,7 +17,7 @@
 #        by * none
 
 # Space separated list of LDAP hosts to use. host:port is allowed too.
-#hosts =
+hosts = localhost
 
 # LDAP URIs to use. You can use this instead of hosts list. Note that this
 # setting isn't supported by all LDAP libraries.
@@ -25,10 +25,10 @@
 
 # Distinguished Name - the username used to login to the LDAP server.
 # Leave it commented out to bind anonymously (useful with auth_bind=yes).
-#dn = 
+dn = cn=dovecot,ou=users,ou=system,ou=directory
 
 # Password for LDAP server, if dn is specified.
-#dnpass = 
+dnpass = SomeThing@1234
 
 # Use SASL binding instead of the simple binding. Note that this changes
 # ldap_version automatically to be 3 if it's lower.
@@ -42,7 +42,7 @@
 #sasl_authz_id =
 
 # Use TLS to connect to the LDAP server.
-#tls = no
+tls = yes
 # TLS options, currently supported only with OpenLDAP:
 #tls_ca_cert_file =
 #tls_ca_cert_dir =
@@ -66,7 +66,7 @@
 # The pass_filter is used to find the DN for the user. Note that the pass_attrs
 # is still used, only the password field is ignored in it. Before doing any
 # search, the binding is switched back to the default DN.
-#auth_bind = no
+auth_bind = yes
 
 # If authentication binding is used, you can save one LDAP request per login
 # if users' DN can be specified with a common template. The template can use
@@ -82,20 +82,33 @@
 # For example:
 #   auth_bind_userdn = cn=%u,ou=people,o=org
 #
-#auth_bind_userdn =
+# Not using this after all, because "If you're using DN template, pass_attrs
+# and pass_filter settings are completely ignored."
+# See https://wiki2.dovecot.org/AuthDatabase/LDAP/AuthBinds
+#auth_bind_userdn = uid=%n,ou=users,dc=%d,ou=directory
 
 # LDAP protocol version to use. Likely 2 or 3.
-#ldap_version = 3
+ldap_version = 3
 
 # LDAP base. %variables can be used here.
 # For example: dc=mail, dc=example, dc=org
-base =
+# Theoretically, we could use something rather specific like:
+# base = ou=users,dc=%d,ou=directory
+# except for the fact that it won't work with the iterate_filter, which
+# prevents us from using the administrative commands from doveadm that
+# act on all mailboxes. So, instead, we intentionally use a very high
+# level base DN which doesn't require any %variables.
+# As a consequence of that decision, we have to include the (mail=%u)
+# predicate in *_filter settings down below; if we could use the
+# per-domain base DN, then (uid=%n) would be sufficient, and we could
+# replace mail=user for *_attrs settings with uid=username.
+base = ou=directory
 
 # Dereference: never, searching, finding, always
-#deref = never
+deref = never
 
 # Search scope: base, onelevel, subtree
-#scope = subtree
+scope = subtree
 
 # User attributes are given in LDAP-name=dovecot-internal-name list. The
 # internal names are:
@@ -106,14 +119,14 @@
 #
 # There are also other special fields which can be returned, see
 # http://wiki2.dovecot.org/UserDatabase/ExtraFields
-#user_attrs = homeDirectory=home,uidNumber=uid,gidNumber=gid
+user_attrs = mail=user
 
 # Filter for user lookup. Some variables can be used (see
 # http://wiki2.dovecot.org/Variables for full list):
 #   %u - username
 #   %n - user part in user@domain, same as %u if there's no domain
 #   %d - domain part in user@domain, empty if user there's no domain
-#user_filter = (&(objectClass=posixAccount)(uid=%u))
+user_filter = (&(objectClass=virtualMailRecipient)(virtualMailboxEnabled=TRUE)(uid=%n)(mail=%u))
 
 # Password checking attributes:
 #  user: Virtual user name (user@domain), if you wish to change the
@@ -121,7 +134,9 @@
 #  password: Password, may optionally start with {type}, eg. {crypt}
 # There are also other special fields which can be returned, see
 # http://wiki2.dovecot.org/PasswordDatabase/ExtraFields
-#pass_attrs = uid=user,userPassword=password
+# Intentionally not fetching password, because we use authentication
+# binds. See https://wiki2.dovecot.org/AuthDatabase/LDAP/AuthBinds
+pass_attrs = mail=user
 
 # If you wish to avoid two LDAP lookups (passdb + userdb), you can use
 # userdb prefetch instead of userdb ldap in dovecot.conf. In that case you'll
@@ -131,11 +146,11 @@
 #  homeDirectory=userdb_home,uidNumber=userdb_uid,gidNumber=userdb_gid
 
 # Filter for password lookups
-#pass_filter = (&(objectClass=posixAccount)(uid=%u))
+pass_filter = (&(objectClass=virtualMailRecipient)(|(virtualMailboxEnabled=TRUE)(virtualMailAuthenticatedSender=TRUE))(uid=%n)(mail=%u))
 
 # Attributes and filter to get a list of all users
-#iterate_attrs = uid=user
-#iterate_filter = (objectClass=posixAccount)
+iterate_attrs = mail=user
+iterate_filter = (&(objectClass=virtualMailRecipient)(virtualMailboxEnabled=TRUE))
 
 # Default password scheme. "{scheme}" before password overrides this.
 # List of supported schemes is in: http://wiki2.dovecot.org/Authentication
diff -ru a/dovecot.conf b/dovecot.conf
--- a/dovecot.conf	2020-01-06 07:54:20.554895000 -0800
+++ b/dovecot.conf	2020-01-17 22:33:41.830801000 -0800
@@ -21,7 +21,7 @@
 # --sysconfdir=/usr/local/etc --localstatedir=/var
 
 # Protocols we want to be serving.
-#protocols = imap pop3 lmtp submission
+protocols = imap lmtp
 
 # A comma separated list of IPs or hosts where to listen in for connections. 
 # "*" listens in all IPv4 interfaces, "::" listens in all IPv6 interfaces.
