diff -ru a/sshd b/sshd
--- a/sshd	2023-12-14 13:46:21.703835000 -0800
+++ b/sshd	2023-12-14 13:49:06.968650000 -0800
@@ -6,7 +6,7 @@
 # auth
 #auth		sufficient	pam_krb5.so		no_warn try_first_pass
 #auth		sufficient	pam_ssh.so		no_warn try_first_pass
-auth		required	pam_unix.so		no_warn try_first_pass
+auth		required	pam_unix.so		no_warn try_first_pass authtok_prompt="%u@%h's password: "
 
 # account
 account		required	pam_nologin.so
