--- a/sshd	2016-08-13 05:14:35.920095000 -0700
+++ b/sshd	2016-08-12 06:28:12.056896000 -0700
@@ -9,7 +9,7 @@
 auth		requisite	pam_opieaccess.so	no_warn allow_local
 #auth		sufficient	pam_krb5.so		no_warn try_first_pass
 #auth		sufficient	pam_ssh.so		no_warn try_first_pass
-auth		required	pam_unix.so		no_warn try_first_pass
+auth		required	pam_unix.so		no_warn try_first_pass authtok_prompt="%u@%h's password: "
 
 # account
 account		required	pam_nologin.so
