diff -ru a/system b/system
--- a/system	2023-12-14 13:46:21.669237000 -0800
+++ b/system	2023-12-14 13:49:06.935393000 -0800
@@ -6,7 +6,7 @@
 # auth
 #auth		sufficient	pam_krb5.so		no_warn try_first_pass
 #auth		sufficient	pam_ssh.so		no_warn try_first_pass
-auth		required	pam_unix.so		no_warn try_first_pass nullok
+auth		required	pam_unix.so		no_warn try_first_pass nullok authtok_prompt="Password: "
 
 # account
 #account	required	pam_krb5.so
