diff -ru a/php-fpm.d/www.conf b/php-fpm.d/www.conf
--- a/php-fpm.d/www.conf	2024-02-04 21:45:32.854844000 -0800
+++ b/php-fpm.d/www.conf	2024-02-04 22:09:53.919295000 -0800
@@ -449,7 +449,7 @@
 ; Setting to "no" will make all environment variables available to PHP code
 ; via getenv(), $_ENV and $_SERVER.
 ; Default Value: yes
-;clear_env = no
+clear_env = no
 
 ; Limits the extensions of the main script FPM will allow to parse. This can
 ; prevent configuration mistakes on the web server side. You should only limit
@@ -463,7 +463,7 @@
 ; the current environment.
 ; Default Value: clean env
 ;env[HOSTNAME] = $HOSTNAME
-env[PATH] = /usr/local/bin:/usr/bin:/bin
+env[PATH] = /opt/local/bin:/usr/local/bin:/usr/local/sbin:/bin:/usr/bin:/usr/sbin:/sbin:/usr/games
 env[TMP] = /tmp
 env[TMPDIR] = /tmp
 env[TEMP] = /tmp
