diff -ru a/php-fpm.d/www.conf b/php-fpm.d/www.conf
--- a/php-fpm.d/www.conf	2021-06-02 18:20:16.000000000 -0700
+++ b/php-fpm.d/www.conf	2021-07-04 10:32:12.305566000 -0700
@@ -37,7 +37,7 @@
 ;                                  FreeBSD for security reasons;
 ;   '/path/to/unix/socket' - to listen on a unix socket.
 ; Note: This value is mandatory.
-listen = 127.0.0.1:9000
+listen = /var/run/php-fpm.sock
 
 ; Set listen(2) backlog.
 ; Default Value: 511 (-1 on FreeBSD and OpenBSD)
@@ -48,9 +48,9 @@
 ; BSD-derived systems allow connections regardless of permissions.
 ; Default Values: user and group are set as the running user
 ;                 mode is set to 0660
-;listen.owner = www
-;listen.group = www
-;listen.mode = 0660
+listen.owner = www
+listen.group = www
+listen.mode = 0660
 ; When POSIX Access Control Lists are supported you can set them using
 ; these options, value is a comma separated list of user/group names.
 ; When set, listen.owner and listen.group are ignored
