diff -ru a/php.ini b/php.ini
--- a/php.ini	2022-04-01 23:40:24.000000000 -0700
+++ b/php.ini	2022-04-06 19:32:09.007155000 -0700
@@ -795,7 +795,7 @@
 ; of zero causes PHP to behave as before.  Default is 1.  You should fix your scripts
 ; to use SCRIPT_FILENAME rather than PATH_TRANSLATED.
 ; http://php.net/cgi.fix-pathinfo
-;cgi.fix_pathinfo=1
+cgi.fix_pathinfo=0
 
 ; if cgi.discard_path is enabled, the PHP CGI binary can safely be placed outside
 ; of the web tree and people will not be able to circumvent .htaccess security.
