diff -ru a/php.ini b/php.ini
--- a/php.ini	2023-04-12 16:46:22.800028000 -0700
+++ b/php.ini	2023-04-12 16:48:10.433771000 -0700
@@ -804,7 +804,7 @@
 ; of zero causes PHP to behave as before.  Default is 1.  You should fix your scripts
 ; to use SCRIPT_FILENAME rather than PATH_TRANSLATED.
 ; https://php.net/cgi.fix-pathinfo
-;cgi.fix_pathinfo=1
+cgi.fix_pathinfo=0
 
 ; if cgi.discard_path is enabled, the PHP CGI binary can safely be placed outside
 ; of the web tree and people will not be able to circumvent .htaccess security.
